shoop/front/models/stored_basket.py

```diff
@@ -0,0 +1,41 @@
+# -*- coding: utf-8 -*-
+# This file is part of Shoop.
+#
+# Copyright (c) 2012-2015, Shoop Ltd. All rights reserved.
+#
+# This source code is licensed under the AGPLv3 license found in the
+# LICENSE file in the root directory of this source tree.
+
+from django.conf import settings
+from django.db import models
+from shoop.core.fields import MoneyField, TaggedJSONField
+import random
+
+ID_KEYSPACE = "0123456789abcdefghijklmnopqrstuvwxyz"
+
+
+def generate_key():
+    return "".join(random.choice(ID_KEYSPACE) for x in range(32))
+
+
+class StoredBasket(models.Model):
+    # A combination of the PK and key is used to retrieve a basket for session situations.
+    key = models.CharField(max_length=32, default=generate_key)
```

Comment:
Just checking: If the random source is good, even max_length=19 (> 98 bits) would be enough to to store 1000 baskets per second for 100 years (total 3.1536e12  baskets) with collision probability less than 0.1% (see https://en.wikipedia.org/wiki/Birthday_problem#Probability_table), so we should be safe then... But it is a big if.

I'm just not sure if using `random.choice` will result in good enough randomness (since it uses a PRNG) or should we use `os.urandom` or something similar. What's the purpose of basket key anyway? Preventing access to someone else's basket? I'm worried that it's possible to determine the state of the PRNG from a few basket keys and then all the following basket keys are predetermined with that state info...